jcache/commons/src/main/java/org/infinispan/jcache/Expiration.java

```diff
@@ -35,19 +35,22 @@ public static Duration getExpiry(ExpiryPolicy policy, Operation op) {
             try {
                return policy.getExpiryForCreation();
             } catch (Throwable t) {
+               log.log(Level.WARNING, "Exception while getting expiry for creation. Fallback to default duration eternal.", t);
```

Comment:
All messages on the info level and above need to be localized; please see the `default:` case for example (that one throws localized exception, but plain messages are not so different).